snappi_ixnetwork/vport.py

```diff
@@ -698,8 +698,9 @@ def results(self, request):
             port_row = dict()
             self._set_result_value(port_row, 'name', vport.get('name'))
             location = vport.get('location')
-            if vport.get('connectionState').\
-                startswith('connectedLink') is True:
+            if vport.get(
+                'connectionState'
+            ).startswith('connectedLink') is True:
                 location += ';connected'
             elif len(location) > 0:
```

Comment:
put this code within exception handling block
self._set_result_value(port_row, ext_name, row_val, typ)

Reply:
I did not get it. This is old code I just indented as per flake8.
point me to the detail

Reply:
I am talking about this code which present earlier:

```python
                    for ext_name, int_name, typ in self._RESULT_COLUMNS:
                        try:
                            self._set_result_value(port_row, ext_name,
                                                   row[int_name], typ)
                        except Exception:
                            # TODO print a warning maybe ?
                            pass
```